unlock-app/src/utils/strings.tsx

```diff
@@ -37,3 +39,11 @@ export const getValidNumber = (value: string | number): number | undefined => {
     ? parseInt(`${value}`)
     : undefined
 }
+
+/** Check if string is an address or ens */
+
+export const isAddressOrEns = (address = '') => {
```

Comment:
Let's just export the other one so we don't duplicate logic?

Reply:
sure, you mean from the component or the other in codebase?

Reply:
From the component. And if we use in either places just make sure we only have a single way of doing this.